src/components/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { PageNavLink } from './PageNavLink';
+
+export const Header: React.FC = () => {
```

Comment:
You can do like that
```suggestion
import { FC } from 'react';
import { PageNavLink } from './PageNavLink';

export const Header: FC = () => {
```